torch/_dynamo/variables/builtin.py

```diff
@@ -1071,31 +1073,25 @@ def call_map(self, tx, fn, seq):
             items = [fn.call_function(tx, [x], {}) for x in seq.unpack_var_sequence(tx)]
             return variables.TupleVariable(items)
 
-    def call_sum(self, tx, seq, **kwargs):
+    def call_sum(self, tx, seq, start=_SENTINEL):
         # Special case for sum on tuple of floats and ints
-        if (
-            isinstance(seq, (variables.ListVariable, variables.TupleVariable))
-            and all(
-                isinstance(x, variables.ConstantVariable)
-                and isinstance(x.value, (int, float))
-                for x in seq.items
-            )
-            and not kwargs
+        if isinstance(seq, (variables.ListVariable, variables.TupleVariable)) and all(
+            isinstance(x, variables.ConstantVariable)
+            and isinstance(x.value, (int, float))
+            for x in seq.items
         ):
             new_list = [x.value for x in seq.items]
             return variables.ConstantVariable.create(sum(new_list))
         if seq.has_unpack_var_sequence(tx):
-            start = kwargs.pop(
-                "start", variables.ConstantVariable.create(0)
-            ).as_python_constant()
-            assert not kwargs
+            if start is self._SENTINEL:
+                start = variables.ConstantVariable.create(0)
```

Comment:
Hold on, shouldn't this be... 
```suggestion
            if start is self._SENTINEL:
                start = variables.ConstantVariable.create(0)
            else:
                assert isinstance(start, (int, float))
                start = variables.ConstantVariable.create(start)
```